win32/Makefile

```diff
@@ -1325,12 +1325,12 @@ test-prep : all utils ../pod/perltoc.pod
 
 test : test-prep
 	cd ..\t
```

Comment:
`win32/GNUmakefile` also needs to be updated (`win32/Makefile` is used by `nmake` part of MSVC, `win32/GNUmakefile` is used by GNU make)

Otherwise it looks fine.

Reply:
Added this too.